### PR TITLE
change: [UIE-7563] - Delete tooltip displaying for current plan in Databases

### PR DESCRIPTION
**Pull request by mpolotsk-akamai**
## Description 📝
Delete "Resizing to smaller plans is not supported" tooltip for current plan in Dedicated tab.

## Changes  🔄
Delete "Resizing to smaller plans is not supported" tooltip for current plan in Dedicated tab. 

## Target release date 🗓️
Release date 04/15/24

## Preview 📷

| Before  | After   |
| ------- | ------- |
|  | |

### Verification steps
(How to verify changes)

- Navigate to Databases
- Choose a Database cluster with Dedicated plan
- Navigate to Resize tab
- In the table of plans, the current plan does not have a tooltip.

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/components/PlansPanel/PlanSelection.tsx [hunk 1]
```diff
@@ -106,12 +108,13 @@ export const PlanSelection = (props: PlanSelectionProps) => {
...
   const diskSize = selectedDiskSize ? selectedDiskSize : 0;
   const planTooSmall = diskSize > type.disk;
+  const isSamePlan = type.heading === currentPlanHeading;
   const tooltip = getToolTip({
-    sizeTooSmall: planTooSmall,
-    planIsDisabled: planIsDisabled,
     disabledToolTip: disabledToolTip,
+    isSamePlan: isSamePlan,
+    planIsDisabled: planIsDisabled,
+    sizeTooSmall: planTooSmall,
```
Comment (hana-linode) [inline packages/manager/src/features/components/PlansPanel/PlanSelection.tsx:116]: We can shorthand this
```suggestion
    disabledToolTip,
    isSamePlan,
    planIsDisabled,
    sizeTooSmall: planTooSmall,
```

File: packages/manager/.changeset/pr-10364-changed-1712674027336.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Changed
+---
+
+delete tooltip for current plan in Database Cluster ([#10364](https://github.com/linode/manager/pull/10364))
```
Comment (hana-linode) [inline packages/manager/.changeset/pr-10364-changed-1712674027336.md:5]:
```suggestion
Delete tooltip for current plan in Database Cluster ([#10364](https://github.com/linode/manager/pull/10364))
```

**Review by hana-linode — APPROVED** [review]

**Review by hkhalil-akamai — COMMENTED** [review]
I did some deeper investigation and found the source of the inconsistency to be [DatabaseResize.tsx:208](https://github.com/linode/manager/blob/9b32652137d52b4027f6992d42f9e2c073ae72ba/packages/manager/src/features/Databases/DatabaseDetail/DatabaseResize/DatabaseResize.tsx#L208). I confirmed that simply deleting the line resolves the bug.

However, I'm not sure why this logic exists in the first place and if deleting it will have other impacts. Any knowledge on this?

**Comment by mpolotsk-akamai** [comment]
Thank you @hkhalil-akamai,
I deleted the line at DatabaseResize.tsx:208 and tested it out. Everything seems to be working fine.

**Review by mjac0bs — APPROVED** [review]
Thanks @mpolotsk-akamai! Did not see any regressions and confirmed that the tooltip is not visible for the current Dedicated plan. I will go ahead and merge this once tests finish running and get it into our 4/15 release.

Screenshot 2024-04-10 at 8 53 35 AM

**Pull request merged**